snooty/rstparser.py

```diff
@@ -1072,6 +1072,32 @@ def register_spec_with_docutils(
         )
         builder.add_directive(name, DocutilsDirective)
 
+    # Define tabsets
+    for name in spec.tabs:
+        tabs_base_class: Any = BaseTabsDirective
+        directive = specparser.Directive(
+            inherit=None,
+            help="help with tabset",
```

Comment:
This seems like a placeholder, is there an opportunity to provide a more useful help message?